python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -48,34 +48,32 @@ def _build_fil_classifier(m, data, arg={}, tmpdir=None):
     else:
         raise ImportError("No XGBoost package found")
 
-    # use maximum 1e5 rows to train the model
-    train_size = min(data[0].shape[0], 100000)
+    train_size = data[0].shape[0]
 
     if isinstance(data[0], (pd.DataFrame, pd.Series)):
         train_data = datagen._convert_to_numpy(data[0])[:train_size, :]
         train_label = datagen._convert_to_numpy(data[1])[:train_size]
-        dtrain = xgb.DMatrix(train_data, label=train_label)
     elif isinstance(data[0], np.ndarray):
         train_data = data[0][:train_size, :]
         train_label = data[1][:train_size]
-        dtrain = xgb.DMatrix(train_data, label=train_label)
     elif isinstance(data[0], cudf.DataFrame):
-        train_data_np_ = data[0].as_gpu_matrix().copy_to_host()
-        train_label_np_ = data[1].to_gpu_array().copy_to_host()
-        train_data_np = train_data_np_[:train_size, :]
-        train_label_np = train_label_np_[:train_size]
-        dtrain = xgb.DMatrix(train_data_np, label=train_label_np)
+        train_data_ = data[0].as_gpu_matrix().copy_to_host()
+        train_label_ = data[1].to_gpu_array().copy_to_host()
+        train_data = train_data_[:train_size, :]
```

Comment:
Now that the train size is no longer limited to 1e5 rows, there's no need for extra data truncation.

It also applies to other clauses in this conditional.